eng/pipelines/common/templates/pipeline-with-resources.yml

```diff
@@ -101,7 +101,7 @@ extends:
         image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
 
       browser_wasm:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-webassembly-20230913040940-1edc1c6
+        image: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-webassembly
```

Comment:
is there anything else left in the -webassembly image that we need or can we switch to the plain mariner image?

Reply:
Wasmtime and WASI SDK (which we ignore at the moment)?

Reply:
isn't that for the wasi leg? I think this only deals with browser

Reply:
I have opened https://github.com/dotnet/runtime/pull/102069 to try that. I am not sure whether we still run some tests with v8.